app/src/main/java/com/soen390/conumap/MapsActivity.kt

```diff
@@ -94,24 +338,24 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback, GoogleMap.OnMarker
             map.addMarker(MarkerOptions().
                     position(downTown).
                 title("SGW").icon(BitmapDescriptorFactory.defaultMarker(342.toFloat()))) //sets color, title and position of marker
+            addMarkersToMap()
```

Comment:
@yabdelkawy When you do `map.clear` here it also removes the markers for all the buildings. So I added `addMarkersToMap()` to your function to put them back when toggling campuses. When you do your review could you also make sure that your feature still works as intended?

Reply:
I pulled your feature into mine and they work well in tandem! I don't have any conflicts or issues between the two